helm/sematic-server/templates/ingress.yaml

```diff
@@ -34,6 +35,17 @@ spec:
                 name: {{ $serviceName }}
                 port:
                   number: {{ $servicePort }}
+          {{- if $.Values.deployment.socket_io.dedicated }}
+          - path: {{ printf "%s/socket.io" .path | replace "//" "/" }}
```

Comment:
The paths that we want to direct to the socketio instance all look like `/api/v1/events/`, they don't have `socket.io` in them

Reply:
they do.

```
2023-02-02 18:04:18,966 [INFO] gunicorn.access: 10.42.36.130 - - [02/Feb/2023:18:04:18 +0000] "GET /socket.io/?EIO=4&transport=polling&t=OOJJG1b&sid=rGUmqMwuW5Iw63mnAAAB HTTP/1.1" 200 42 "https://dev2.dev-usw2-sematic0.sematic.cloud/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36"
```